### ISSUE TITLE
Option to include or exclude project evaluation in embedded project

### ISSUE THREAD

**Issue by JetStarBlues** [Khan/live-editor#463]
At the moment, the project evaluation module is included in the `<iframe>` generated by the embed code. 

I think it should be optional whether it appears since on the client side, it is not possible to hide the evaluation module `$(".scratchpad-project-eval-mount")` using css due to cross-origin.

![evalmodule](https://cloud.githubusercontent.com/assets/4354703/10779674/b37a6870-7cf9-11e5-9484-8d887d29c712.png)

**Comment by kevinbarabash** [comment]
Where are you getting the embed code from?  The "Share" button on the site generate embed code which only shows the canvas and optionally the editor and some buttons.

**Comment by JetStarBlues** [comment]
I got the embed code from the "share" button. This happens only for programs that are "projects" such as [Project ad design](https://www.khanacademy.org/computing/computer-programming/programming/text-basics/p/project-ad-design) where evaluation can been requested.

I just realized right now that despite the module always being present, it only has content when you are logged in to khan and it is your project....

So it's not an issue per se. Sorry about that. Thanks
